torch_xla/csrc/debug_util.cpp

```diff
@@ -299,12 +301,11 @@ void DebugUtil::analyze_graph_execution_python_frame(
              : 1)
      << "\n";
 
+  int remain_frame_count = max_frame_count;
   ss << debug_output_prefix << "Python Frame Triggered Execution: \n";
   for (auto& location : frames) {
-    // if current frame `__call__` at pjrt.py, bleow stack will be python
-    // code to spawn up process, not very useful to the user.
-    if (location.function == "__call__" &&
-        endsWith(location.file, "_internal/pjrt.py")) {
+    remain_frame_count--;
+    if (remain_frame_count < 0) {
```

Comment:
Is it possible to just make this a for-i loop like `for (int i = 0; i < max(frames.size, max_frame_count); i++)`?

Reply:
oh right, let me do that in a separate pr so I don't need to rerun the CI lol, I also don't need to backport that into 2.2